shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -790,47 +771,8 @@ - (void)dealloc {
 
 #pragma mark - UIAccessibilityContainer overrides
 
-- (NSInteger)accessibilityElementCount {
-  // This container should only contain 2 elements:
-  // 1. The semantic object that represents this container.
-  // 2. The platform view object.
-  return 2;
-}
-
-- (nullable id)accessibilityElementAtIndex:(NSInteger)index {
-  FML_DCHECK(index < 2);
-  if (index == 0) {
-    return _semanticsObject.nativeAccessibility;
-  } else {
-    return _platformView;
-  }
-}
-
-- (NSInteger)indexOfAccessibilityElement:(id)element {
-  FML_DCHECK(element == _semanticsObject || element == _platformView);
-  if (element == _semanticsObject) {
-    return 0;
-  } else {
-    return 1;
-  }
-}
-
-#pragma mark - UIAccessibilityElement overrides
-
-- (CGRect)accessibilityFrame {
-  return _semanticsObject.accessibilityFrame;
-}
-
-- (BOOL)isAccessibilityElement {
-  return NO;
-}
-
-- (id)accessibilityContainer {
-  return [_semanticsObject accessibilityContainer];
-}
-
-- (BOOL)accessibilityScroll:(UIAccessibilityScrollDirection)direction {
-  return [_platformView accessibilityScroll:direction];
+- (NSArray*)accessibilityElements {
+  return @[ _platformView ];
```

Comment:
you can override the nativeAccessibility prop gettter to return the _platformView